Server/src/index.ts

```diff
@@ -312,17 +312,179 @@ export class CitrineOSServer {
   }
 
   private initAllModules() {
-    [
-      this.getModuleConfig(EventGroup.Certificates),
-      this.getModuleConfig(EventGroup.Configuration),
-      this.getModuleConfig(EventGroup.EVDriver),
-      this.getModuleConfig(EventGroup.Monitoring),
-      this.getModuleConfig(EventGroup.Reporting),
-      this.getModuleConfig(EventGroup.SmartCharging),
-      this.getModuleConfig(EventGroup.Transactions),
-    ].forEach((moduleConfig) => {
-      this.initModule(moduleConfig);
-    });
+
+    if (this._config.modules.certificates) {
```

Comment:
Not sure I really see these changes below as an enhancement.

Seems like we just expanded the for loop. Would've preferred to just keep OCPI related changes instead of doing any other enhancements.

But if I am in the minority, I am fine to proceed with these changes.

Reply:
yeah we expanded the loop because the different module classes need different repository classes passed in during initialization, and previously we werent passing the repos in and so new repos were being created. We could have still done it with the loop I suppose but we wanted to move quickly and this is more explicit I suppose, but I think if we move towards DI then much of this would be adjusted most likely